content/2.blog/2025/05/06/portfolio-redesign.md

```
@@ -0,0 +1,261 @@
+2025年5月上旬、以下の技術スタックで新しいポートフォリオサイトを構築しました：
+
+- **フレームワーク**: Nuxt 3
+- **コンテンツ管理**: @nuxt/content
+- **UIライブラリ**: NuxtUI
+- **スタイリング**: Tailwind CSS v3
+
+### コンテンツ構造の設計
+
+Nuxt Contentの機能を最大限活用するため、以下のようなコンテンツ構造を設計しました：
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)